apps/someone-else/controllers/personal-details.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var util = require('util');
+var Parent = require('../../../lib/date-controller');
```

Comment:
hof's date controller caused a bug when I used it before. I'd like to avoid it and wait until hof's base controller supports dates.